LSR Rebuild

TeamLemon opened the pull request:
Note: Theta_Delta and Theta_epsilon have been changed now to be
consistent with the other theta matrices, i.e. including a M_Theta and
M_epsilon and post-multiplying by LambdaMatrix. The VTheta function now
takes delta_X inputs for _all_ ports/noise sources, and we have changed
the definition of X_delta and X_epsilon from Kirk’s thesis.

X_Delta is
LambdaMat*[Delta_delta_a;Delta_delta_a;Delta_delta_b;Delta_delta_b],

X_epsilon is
LambdaMat*[delta_epsilon;conj(delta_epsilon);delta_epsilon;conj(delta_ep
silon)]

TeamLemon committed:
Create XSSRefl.m

TeamLemon committed:
CommittingThetaMatricesToLSR-Rebuild

Now I will do things in the right place

TeamLemon committed:
Individual Theta functions

TeamLemon committed:
Updated

Commit:
quadRotation: Fixed missing minus signs in rotation matrix

I think the rotation matrix requires that there be a minus for the sign
in the second row of the rotation matrix.  This was missing so I
changed it.

Commit:
Added twice the phase evolution to the pump field

Not sure if these is correct.  Intuitively it seems that the pump phase evolves at twice the rate of the fundamental.  Need to make absolutely clear to the user that this is the assumption so that they don't try to preempt it in the calling function.

Commit:
Big update generalising Variance Calc

This function has been massively modified to now take any number of noise inputs.  Big change in usage, now feed in Theta matrix followed by quadratures X in pairs.  As pairs of these may be entered as the user likes followed by an optional specification of quadrature angle Phi.  This should ensure that the whole model is really expandable.

Commit:
Migrated name of VReflTheta to more generic VTheta

Seems to make sense to drop the reference to reflected port as this is a much more generalised function in the new modification.

TeamLemon committed:
Changes to VTheta and Theta functions test

TeamLemon committed:
Theta_Delta and Theta Epsilon update

To make use of Chi’s and X’s consistent with other Theta functions, now
VRefl can use all X’s all the time

TeamLemon committed:
Fixes to VTheta

Note: Theta_Delta and Theta_epsilon have been changed now to be
consistent with the other theta matrices, i.e. including a M_Theta and
M_epsilon and post-multiplying by LambdaMatrix. The VTheta function now
takes delta_X inputs for *all* ports/noise sources, and we have changed
the definition of X_delta and X_epsilon from Kirk’s thesis.

X_Delta is
LambdaMat*[Delta_delta_a;Delta_delta_a;Delta_delta_b;Delta_delta_b],

X_epsilon is
LambdaMat*[delta_epsilon;conj(delta_epsilon);delta_epsilon;conj(delta_ep
silon)]

fincle commented:
Do you see this reply if I send a reply via email?



> On 20 Nov 2015, at 3:40 PM, TeamLemon <notifications@github.com> wrote:
> 
> Note: Theta_Delta and Theta_epsilon have been changed now to be
> consistent with the other theta matrices, i.e. including a M_Theta and
> M_epsilon and post-multiplying by LambdaMatrix. The VTheta function now
> takes delta_X inputs for all ports/noise sources, and we have changed
> the definition of X_delta and X_epsilon from Kirk’s thesis.
> 
> X_Delta is
> LambdaMat*[Delta_delta_a;Delta_delta_a;Delta_delta_b;Delta_delta_b],
> 
> X_epsilon is
> LambdaMat*[delta_epsilon;conj(delta_epsilon);delta_epsilon;conj(delta_ep
> silon)]
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/fincle/ANUVOPONoiseModel/pull/2 <https://github.com/fincle/ANUVOPONoiseModel/pull/2>
> Commit Summary
> 
> Create XSSRefl.m
> CommittingThetaMatricesToLSR-Rebuild
> Individual Theta functions
> Updated
> quadRotation: Fixed missing minus signs in rotation matrix
> Added twice the phase evolution to the pump field
> Big update generalising Variance Calc
> Migrated name of VReflTheta to more generic VTheta
> Changes to VTheta and Theta functions test
> Theta_Delta and Theta Epsilon update
> Fixes to VTheta
> File Changes
> 
> A THETAReflPhi.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-0> (89)
> A THETA_Delta.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-1> (70)
> A THETA_Delta.m~ <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-2> (70)
> A THETA_epsilon.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-3> (70)
> A THETA_in.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-4> (60)
> A THETA_loss.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-5> (60)
> A THETA_out.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-6> (59)
> A VOPONoiseBudget_v1_1CavityLengthNoiseTest.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-7> (145)
> A VOPONoiseBudget_v1_1ThetaFunctionsTest.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-8> (169)
> A VOPONoiseBudget_v1_1ThetaFunctionsTest.m~ <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-9> (169)
> A VOPONoiseBudget_v2_1ThetaFunctionsTest.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-10> (181)
> A VTheta.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-11> (88)
> A VTheta.m~ <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-12> (88)
> A XSSRefl.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-13> (123)
> A cavityLengthNoise.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-14> (33)
> A quadRotation.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-15> (7)
> A rms.m <https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-16> (17)
> Patch Links:
> 
> https://github.com/fincle/ANUVOPONoiseModel/pull/2.patch <https://github.com/fincle/ANUVOPONoiseModel/pull/2.patch>
> https://github.com/fincle/ANUVOPONoiseModel/pull/2.diff <https://github.com/fincle/ANUVOPONoiseModel/pull/2.diff>
> —
> Reply to this email directly or view it on GitHub <https://github.com/fincle/ANUVOPONoiseModel/pull/2>.
>

TeamLemon commented:
jajajajajjajaja

On Fri, Nov 20, 2015 at 5:07 PM, Andrew Wade <notifications@github.com>
wrote:

> Do you see this reply if I send a reply via email?
>
>
>
> > On 20 Nov 2015, at 3:40 PM, TeamLemon <notifications@github.com> wrote:
> >
> > Note: Theta_Delta and Theta_epsilon have been changed now to be
> > consistent with the other theta matrices, i.e. including a M_Theta and
> > M_epsilon and post-multiplying by LambdaMatrix. The VTheta function now
> > takes delta_X inputs for all ports/noise sources, and we have changed
> > the definition of X_delta and X_epsilon from Kirk’s thesis.
> >
> > X_Delta is
> > LambdaMat*[Delta_delta_a;Delta_delta_a;Delta_delta_b;Delta_delta_b],
> >
> > X_epsilon is
> > LambdaMat*[delta_epsilon;conj(delta_epsilon);delta_epsilon;conj(delta_ep
> > silon)]
> >
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/fincle/ANUVOPONoiseModel/pull/2 <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2>
> > Commit Summary
> >
> > Create XSSRefl.m
> > CommittingThetaMatricesToLSR-Rebuild
> > Individual Theta functions
> > Updated
> > quadRotation: Fixed missing minus signs in rotation matrix
> > Added twice the phase evolution to the pump field
> > Big update generalising Variance Calc
> > Migrated name of VReflTheta to more generic VTheta
> > Changes to VTheta and Theta functions test
> > Theta_Delta and Theta Epsilon update
> > Fixes to VTheta
> > File Changes
> >
> > A THETAReflPhi.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-0> (89)
> > A THETA_Delta.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-1> (70)
> > A THETA_Delta.m~ <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-2> (70)
> > A THETA_epsilon.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-3> (70)
> > A THETA_in.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-4> (60)
> > A THETA_loss.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-5> (60)
> > A THETA_out.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-6> (59)
> > A VOPONoiseBudget_v1_1CavityLengthNoiseTest.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-7> (145)
> > A VOPONoiseBudget_v1_1ThetaFunctionsTest.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-8> (169)
> > A VOPONoiseBudget_v1_1ThetaFunctionsTest.m~ <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-9> (169)
> > A VOPONoiseBudget_v2_1ThetaFunctionsTest.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-10> (181)
> > A VTheta.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-11> (88)
> > A VTheta.m~ <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-12> (88)
> > A XSSRefl.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-13> (123)
> > A cavityLengthNoise.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-14> (33)
> > A quadRotation.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-15> (7)
> > A rms.m <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2/files#diff-16> (17)
> > Patch Links:
> >
> > https://github.com/fincle/ANUVOPONoiseModel/pull/2.patch <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2.patch>
> > https://github.com/fincle/ANUVOPONoiseModel/pull/2.diff <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2.diff>
> > —
> > Reply to this email directly or view it on GitHub <
> https://github.com/fincle/ANUVOPONoiseModel/pull/2>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fincle/ANUVOPONoiseModel/pull/2#issuecomment-158292288>
> .
>
